pages/services/prometheus/0.1.0-2.3.1/connecting-clients/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: layout.pug
+navigationTitle:  Connecting Clients
+title: Connecting Clients
+menuWeight: 70
+excerpt: Connecting clients through service discovery
+featureMaturity:
+enterprise: false
+---
+
+# Connecting Clients
+One of the benefits of running containerized services is that they can be placed anywhere in the cluster. Since they can be deployed anywhere on the cluster, clients need a way to find the service. This is where service discovery comes in.
+
+
+## Discovering Endpoints
+
+Once the service is running, you may view information about its endpoints via either of the following methods:
+- CLI:
+  1. List endpoint types: `dcos prometheus endpoints`
+  2. View endpoints for an endpoint type: `dcos prometheus endpoints <endpoint>`
+- API:
+  1. List endpoint types: `<dcos-url>/service/prometheus/v1/endpoints`
+  2. View endpoints for an endpoint type: `<dcos-url>/service/prometheus/v1/endpoints/<endpoint>`
+
+Returned endpoints will include the following:
+- `.autoip.dcos.thisdcos.directory` hostnames for each instance that will follow them if they're moved within the DC/OS cluster.
+- A HA-enabled VIP hostname for accessing any of the instances (optional).
+- A direct IP address for accessing the service if `.autoip.dcos.thisdcos.directory` hostnames are not resolvable.
+- If your service is on a virtual network such as the `dcos` overlay network, then the IP will be from the subnet allocated to the host that the task is running on. It will not be the host IP. To resolve the host IP use Mesos DNS (`<task>.<service>.mesos`).
+
+In general, the `.autoip.dcos.thisdcos.directory` endpoints will only work from within the same DC/OS cluster. From outside the cluster you can either use direct IPs or set up a proxy service that acts as a frontend to your prometheus instance. For development and testing purposes, you can use [DC/OS Tunnel](https://docs.mesosphere.com/1.10/administering-clusters/sshcluster/) to access services from outside the cluster, but this option is not suitable for production use.
+
+
+## Connection Response
+
+The response, for both the CLI and the REST API is as below.
+
+```shell
```

Comment:
this doesnt look like the prometheus example

Reply:
Updated